web3/utils/toolz/__init__.py

```diff
@@ -4,6 +4,7 @@
         complement,
         compose,
         concat,
+        cons,
```

Comment:
Hm, maybe it's time to bump the eth-utils requirement up and start using `eth_utils.toolz`